src/directives.ts

```diff
@@ -0,0 +1,121 @@
+import { VNode, VueConstructor } from 'vue';
+import { DirectiveBinding, DirectiveOptions } from 'vue/types/options';
+
+import { getClosest } from './utils';
+import { pluginOptions } from './main';
+
+export type trackClickHandlerFunction = (eventName: string | undefined, eventProps: {[key: string]: any}) => void; // eslint-disable-line @propelinc/no-explicit-any
```

Comment:
This definition sort of sticks out here compared to the other functions in the PluginOptions object. Should we just move it inline to be consistent?

Reply:
good idea